### DISCUSSION TITLE
What is the name of the dot above a lowercase i and j?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#548] [Galaxy Trivia] [answered]
These weird and wacky marks are called "diacritics"

**Comment by peterbaccaro** [comment]
It's called a **tittle** and comes from the Latin phrase titulus, which means heading or inscription.

**Comment by Veercodeprog** [comment, marked as answer]
it is called tittle.

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [Galaxy Brain](https://github.com/Veercodeprog?tab=achievements&achievement=galaxy-brain) badge. Keep up the great work and unlock the Bronze 🥉 badge next by answering 8 questions. Don't forget to post your own questions too! 😊🎉

**Reply by peterbaccaro** [reply]
That's actually the wrong answer, the correct answer is tittle.

**Reply by ParanoidUser** [reply]
Well spotted! We appreciate you giving the answer 'semicolonoscopy' it needed! 😉

**Reply by peterbaccaro** [reply]
🤓🤯😵🤪😝

**Comment by thuongtruong1009** [comment]
The dot above the lowercase letters "i" and "j" is called a "tittle". It is a small distinguishing mark, or diacritic, used in many alphabets that use the Latin, Cyrillic, and Arabic scripts, among others. In the case of the letter "i" and "j", the tittle serves to differentiate them from other letters that might look similar, such as the lowercase "l" (ell) in some fonts.
